source/codegen/metadata/nifake_extension/enums.py

```diff
@@ -1,3 +1,20 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-FAKE API metadata version 1.2.0d9
-enums = {}
+enums = {
```

Comment:
Putting this in `nifake` would be better, assuming there's not a downside to that I'm missing. `nifake_extension` is intended to prove we can support a service that operates on sessions from a different service. So it doesn't need any new functionality unless it's related somehow to cross-driver sessions. `nifake` is a better home for covering general codegen features.

Reply:
Since I'm reverting this enum file anyway, I'll consider it for future changes. Ryan felt the other change (to functions) was reasonable to do here.